### ISSUE TITLE
Debuild rules incompatible bdb

### ISSUE THREAD

**Issue by linalouise9** [peercoin/peercoin#330]
The debian build is using the flag --with-incompatible-bdb
This means the deb package wallet is not backwards compatible

Instead should somehow use the contrib/install_db4.sh to compile bdb as in scripts/install-ubuntu.sh

**Comment by peerchemist** [comment]
I don't think we should keep compatibility with ancient libdb4.8, Debian has dropped it from it's repositories and only source for it now is Bitcoin-core PPA.
Most of our linux installs use current libdb found in distos repositories.

**Comment by sandakersmann** [comment]
> I don't think we should keep compatibility with ancient libdb4.8

The problem with this is that old wallet files will be unusable. It will gives us an onslaught of support requests.

**Comment by peerchemist** [comment]
We had Linux packages in the past too, and for longer timeframes. My package in ArchLinux AUR is there for about three years. We also had debian/ubuntu repositories in the late v0.5 era without any problems.

**Comment by sandakersmann** [comment]
Maybe Linux users are a bit more tech-savvy?

**Comment by peerchemist** [comment]
> Maybe Linux users are a bit more tech-savvy?

Surely, and this is why I think we should not bother with extreme backward compatibility.

**Comment by sandakersmann** [comment]
@hrobeers @sigmike @backpacker69 What do you guys think?
